tasks/expanse/lib/client.rb

```diff
@@ -67,9 +67,9 @@ def exposures(max_pages = 100, limit_per_page = 10_000)
         end
 
         def cloud_exposure_counts
-          url = "https://expander.expanse.co/api/v1/summaries/cloud/counts"
-          response_body = RestClient.get(url, @headers)
-          result = JSON.parse(response_body)["data"]
+          # url = "https://expander.expanse.co/api/v1/summaries/cloud/counts"
+          # response_body = RestClient.get(url, @headers)
+          # result = JSON.parse(response_body)["data"]
```

Comment:
do we not need this method? looks like the entire thing is being commented out?

Reply:
At some point, we should delete all the unused lines but I just commented them out.